client/components/ProjectHeading.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components/native';
+import { AntDesign } from '@expo/vector-icons';
```

Comment:
would consider just having our own SVG icons instead of relying on a library. Its likely we'd want our own anyway.